synthclient/manage-exemptions/add-ets

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# This script expects to be called as follows:
+#
+#     cat order.json                       \
+#       | add-ets /path/to/customer/data/  \
+#       > order-with-ets.json
+#
+# The input, `order.json`, should be a synthclient request:
+#
+#     {"region": "us", "fasta": "ACTGACTG"}
+#
+# The supplied customer data folder should be laid out like:
+#
+#     foo-2025-01-01.et
+#     foo-2025-01-01.totp
+#     foo-2025-02-02.et
+#     foo-2025-02-02.totp
+#
+# The result, `order-with-ets.json`, will have a new "ets" field
+# ready to be sent to synthclient.
+
+set -e
+
+die() { echo -e "$1" && exit 1; }
+
+command -v jq &> /dev/null || die "jq could not be found. Please install it:\n\n    https://jqlang.org/\n"
+command -v totp-cli &> /dev/null || die "totp-cli could not be found. Please install it:\n\n    https://github.com/yitsushi/totp-cli\n"
+[ "$#" -eq 1 ] || die "Usage:  cat order.json | add-ets /path/to/customer/data"
```

Comment:
Did you used to be a Perl programmer? :)  (It's the check-and-die paradigm. :)  But anyway, this is fine.